app/controllers/notes_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class NotesController < ApplicationController
+  before_action :load_note, only: %i[show edit update]
```

Comment:
Minor thing for consistency, we have usually called this type of method `set_x`, e.g. `set_contact`, `set_need`

Reply:
Thanks!